### ISSUE TITLE
Empty .json files might be skipped instead of making reporter crash

### ISSUE THREAD

**Issue by lenntt** [wswebcreation/multiple-cucumber-html-reporter#47]
While having several cucumbers running, an empty .json file is there.
So when creating the report before one of tests finishes (or after a crash), the reporter crashes too (Unexpected end of JSON input).

Technically an empty string is invalid JSON, so its sort of expected
```
JSON.parse('');
VM59:1 Uncaught SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at <anonymous>:1:6
```

However, a way to ignore empty (non-finalized) files would be very welcome, for my use case where several cucumbers run at the same time.

**Comment by lenntt** [comment]
@wswebcreation if you think this makes sense, I wouldn't mind trying to prepare a PR.

Could put it under a `ignoreEmptyJSON` boolean option, which defaults to false?

**Comment by wswebcreation** [comment]
Hi @lenntt

If you want to make a PR it's highly appreciated!
My suggestion would be to make it default behavior (so no Boolean option). So it's a biggie instead of a feature.

Tnx man!

**Comment by wswebcreation** [comment]
Just released version 1.10.1, tnx man. By the way, let's contact on linkedIn if you like.

**Comment by lenntt** [comment]
excellent, thanks.
you got a linkedin invite from another dutchman :)
